src/client.ts

```diff
@@ -101,30 +130,26 @@ export interface IPageData {
     title: string;
 }
 
-/** One page entry in a layout tree. */
-export interface ITreeEntry {
-    depth: number;
-    route: string;
+/*
+LAYOUT HIERARCHY NODES
```

Comment:
Maybe in a client subdir with an `index.ts` so you can import everthing you need